Client/Frontend/UserContent/UserScripts/RequestBlocking.js

```diff
@@ -0,0 +1,77 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+window.braveBlockRequests = (args) => {
+  "use strict";
+  const securityToken = args.securityToken
```

Comment:
pls include `;`. Javascript parser is not too eager of missing `;`.

https://doc.casthighlight.com/alt_missingsemicolon-always-use-semicolons/